research/flamby/fed_heart_disease/fenda/client.py

```diff
@@ -50,11 +51,11 @@ def __init__(
         self.client_number = client_number
         self.learning_rate: float = learning_rate
         if cos_sim_activate:
-            self.cos_sim_loss_weight = 100.0
+            assert isinstance(extra_loss_weights, float)
```

Comment:
Rather than doing an `isinstance` check here, you can just do `assert extra_loss_weights is not None`.

Reply:
Sure